### ISSUE TITLE
[enhancement] overall progress

### ISSUE THREAD

**Issue by mattia-b89** [rg3/youtube-dl#9634]
### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2016.05.21.2*. If it's not read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2016.05.21.2**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [x] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

### Description of your *issue*, suggested solution and other information

I'd like to have an overall progress bar, an overall complete percentage, when downloading/elaborating more links at once.
e.g. (suggested implementation)
```
$ youtube-dl https://aaaaa https://bbbbb https://ccccc
[youtube] aaaaa: Downloading webpage
[youtube] aaaaa: Downloading video info webpage
[youtube] aaaaa: Extracting video information
[youtube] aaaaa: Downloading MPD manifest
[download] Destination: aaaaa.webm
[download] 100%
[1 of 3]
[youtube] bbbbb: Downloading webpage
[youtube] bbbbb: Downloading video info webpage
[youtube] bbbbb: Extracting video information
[youtube] bbbbb: Downloading MPD manifest
[download] Destination: bbbbb.webm
[download] 100%
[2 of 3]
[youtube] ccccc: Downloading webpage
[youtube] ccccc: Downloading video info webpage
[youtube] ccccc: Extracting video information
[youtube] ccccc: Downloading MPD manifest
[download] Destination: ccccc.webm
[download] 100%
[3 of 3]
```

**Comment by dstftw** [comment]
That is not possible since playlist may contain playlists thus making such progress inaccurate. Moreover for plain playlists with just videos it's already obvious what the progress is.

**Comment by mattia-b89** [comment]
this is the reason why I suggested number of items i.e. [1 of 3] than
percentage....

**Comment by dstftw** [comment]
There are already numbers of items:
```
[download] Downloading playlist: 29.05
[yandexmusic:playlist] playlist 29.05: Collected 44 video ids (downloading 44 of them)
[download] Downloading video 1 of 44
[yandexmusic:track] 6573122: Downloading track JSON
[yandexmusic:track] 6573122: Downloading track location JSON
[info] Available formats for 6573122:
format code  extension  resolution note
0            mp3        unknown    5.78MiB
[download] Downloading video 2 of 44
[yandexmusic:track] 20513268: Downloading track JSON
[yandexmusic:track] 20513268: Downloading track location JSON
[info] Available formats for 20513268:
format code  extension  resolution note
0            mp3        unknown    5.11MiB
[download] Downloading video 3 of 44
[yandexmusic:track] 9019659: Downloading track JSON
[yandexmusic:track] 9019659: Downloading track location JSON
[info] Available formats for 9019659:
format code  extension  resolution note
0            mp3        unknown    6.14MiB
...
```
